docs/source/index.rst

```diff
@@ -1,21 +1,37 @@
 JupyterHub
 ==========
 
-`JupyterHub`_, a multi-user **Hub**, spawns, manages, and proxies multiple
+`JupyterHub`_ is the best way to serve `Jupyter notebook`_ for multiple users. 
+It can be used in a classes of students, a corporate data science group or scientific 
+research group. It is a multi-user **Hub** that spawns, manages, and proxies multiple
 instances of the single-user `Jupyter notebook`_ server.
-JupyterHub can be used to serve notebooks to a class of students, a corporate
-data science group, or a scientific research group.
 
-.. image:: images/jhub-parts.png
-   :alt: JupyterHub subsystems
-   :width: 40%
-   :align: right
+To make life easier, JupyterHub have some pre-configured projects. Be sure to 
```

Comment:
How about calling them 'distributions' instead of 'pre-configured projects'? I think that's the word we use in both TLJH and zero to jupyterhub docs.